docs/tutorials/fundamentals/viewer.md

```diff
@@ -76,81 +57,89 @@ import napari
 viewer = napari.Viewer()
 new_layer = viewer.add_image(data.astronaut(), rgb=True)
 ```
+{meth}[`add_image<napari.components.viewer_model.ViewerModel.add_image>`](https://napari.org/stable/api/napari.components.ViewerModel.html?highlight=add_image#napari.components.ViewerModel.add_image) accepts the same arguments as {func}`imshow<napari.show>` but only returns an {class}`Image<napari.layers.Image>` layer instead of both the {class}`Viewer<napari.Viewer>` and {class}`Image<napari.layers.Image>` layer (as you must already have a viewer to use it).
```

Comment:
```suggestion
{meth}`add_image<napari.components.ViewerModel.add_image>` accepts the same arguments as {func}`imshow<napari.imshow>` but only returns an {class}`Image<napari.layers.Image>` layer instead of both the {class}`Viewer<napari.Viewer>` and {class}`Image<napari.layers.Image>` layer (as you must already have a viewer to use it).
```

Hard-coding the link to the {meth} makes a graphical artifact:
<img width="760" alt="image" src="https://user-images.githubusercontent.com/76622105/211203076-f004f995-2c60-4dad-ac87-04de2e43a641.png">

I think I've fixed both the `{meth}` and `{func}imshow`